views/index.hbs

```diff
@@ -1,8 +1,8 @@
-﻿<h1 class="f1 f-6-ns tc"><a class="white no-underline" href="/">Hacktoberfest Checker</a></h1>
+﻿<h1 class="f1 f-6-ns tc pb4"><a class="white no-underline" href="">Hacktoberfest Checker</a></h1>
 
 <div class="tc pb4">
     <p class="white">{{ timemessage }}</p>
-    <form action="/" class="center h2 w-30-ns w-60-m w-80 flex" method="get">
+    <form action="" class="center h2 w-30-ns w-60-m w-80 flex" method="get">
```

Comment:
Is this needed? I know it probably just submits to the same page, but looks strange with a blank action.

Reply:
This I have to test. I did it because I knew it would work this way.

Reply:
Yes, it seems to work, even though it seems wrong. I will add it back though, since it's your code.